src/_repobee/cli/dispatch.py

```diff
@@ -60,7 +60,8 @@ def dispatch_command(
         plug.cli.CoreCommand.repos.update,
         plug.cli.CoreCommand.repos.clone,
     ]:
-        plug.echo(formatters.format_hook_results_output(hook_results))
+        if hook_results:
+            plug.echo(formatters.format_hook_results_output(hook_results))
```

Comment:
I don't think the outer if is necessary. In fact, I'm pretty sure that it can be removed. In other words, this can be scrapped:

```python
    if is_ext_command or args.action in [
        plug.cli.CoreCommand.repos.setup,
        plug.cli.CoreCommand.repos.update,
        plug.cli.CoreCommand.repos.clone,
    ]:
```

I think. Try it.

Reply:
I mean, if the hook results are non-empty, then we should log them regardless of which command was called.

Reply:
If we remove the outer loop as you say, we ALSO get this result (after the expected result by the command)

``` 

list-issues: SUCCESS
Fetched 2 issues from denj-task-4


hook results for emilsjol-task-4

list-issues: SUCCESS
Fetched 2 issues from emilsjol-task-4


hook results for brynolv-task-4

list-issues: SUCCESS
Fetched 2 issues from brynolv-task-4


hook results for evper-task-4

list-issues: SUCCESS
Fetched 2 issues from evper-task-4


hook results for ofjall-task-4

list-issues: SUCCESS
Fetched 1 issues from ofjall-task-4


hook results for sahlest-task-4

list-issues: SUCCESS
Fetched 2 issues from sahlest-task-4


hook results for lrezza-task-4

list-issues: SUCCESS
Fetched 2 issues from lrezza-task-4


hook results for morrje-task-4

list-issues: SUCCESS
Fetched 2 issues from morrje-task-4


hook results for nodelius-task-4

list-issues: SUCCESS
Fetched 2 issues from nodelius-task-4


hook results for ollindg-task-4

list-issues: SUCCESS
Fetched 2 issues from ollindg-task-4


hook results for hakki-task-4

list-issues: SUCCESS
Fetched 2 issues from hakki-task-4


hook results for kfolke-task-4

list-issues: SUCCESS
Fetched 2 issues from kfolke-task-4


hook results for list-issues

meta: SUCCESS
Meta info about the list-issues hook results
```

Reply:
So, before this PR we had an extra line, the pushed code gives no extra line, and your suggested change gives the previous comment.

All this is of course after the result of the command that i run (Which was `issues list`

Reply:
Okay, then that needs to be fixed separately I suppose. The current nesting of if-statements is unmaintainable, but this new problem is unrelated to your PR.

I'll merge this PR and create a separate issue for removing the nesting. I'm guessing we need to add metainfo to the hook results regarding whether or not they should be printed to stdout/log. It's probably time to seriously think over the format as well.